AccountsManager/src/main/java/di/AccountsManagerModule.java

```diff
@@ -0,0 +1,39 @@
+package di;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.inject.AbstractModule;
+import configuration.MybatisConfiguration;
+import io.logz.guice.jersey.JerseyModule;
+import io.logz.guice.jersey.configuration.JerseyConfiguration;
+import org.apache.ibatis.reflection.factory.DefaultObjectFactory;
+import org.apache.ibatis.reflection.wrapper.DefaultObjectWrapperFactory;
+
+import java.io.File;
+import java.io.IOException;
+
+public class AccountsManagerModule extends AbstractModule {
+
+    public AccountsManagerModule(){
+
+    }
+
+    @Override
+    protected void configure() {
+        ObjectMapper mapper = new ObjectMapper();
```

Comment:
Instead of all `new ObjectMapper()` occurrences, let's have a class `JsonParser` with static methods for parsing/unparsing json strings. In addition, it will convert the checked-exception that Jackson is throwing to an unchecked-exception (`RuntimeException`) - this way we won't have to catch exceptions when parsing jsons